packages/gatsby-source-lever/src/gatsby-node.js

```diff
@@ -3,6 +3,50 @@ const normalize = require(`./normalize`)
 
 const typePrefix = `lever__`
 
+exports.createSchemaCustomization = async ({ actions }) => {
+  const typeDefs = `
+    type lever implements Node {
```

Comment:
One of the goals for schema customization is to opt-out of type inference which is very expensive ([see docs](https://www.gatsbyjs.org/docs/schema-customization/#opting-out-of-type-inference)). So we should do this:
```suggestion
    type lever implements Node @dontInfer {
```